models/project-template-tasks.js

```diff
@@ -67,6 +64,14 @@ module.exports = {
         improvementProjectDetails : {
             type : Object,
             default : {}
+        },
+        visibleIf : {
+            type : Array,
+            default : []
+        },
+        hasSubTasks : {
```

Comment:
@aman-tunerlabs - I think we should have one more key of status (active/inactive) to determine if a task is still to be shown for the template